R/draft_pick.R

```diff
@@ -69,5 +70,5 @@ draft_pick <- function(pick_number = 1, round_number = NULL, year = NULL) {
     break
   }
 
-  df
+  return(df)
```

Comment:
@PolinaRomanchenko I believe the proper syntax in R is to not specify the `return` and instead just to just identify the variable you want returned (i.e. the last variable).


```suggestion
    df
```

Reply:
Yeah, overlooked it. Thank you!